### PR TITLE
modify ddpg test

### PR DESCRIPTION
**Pull request by yangdsh**
Test ddpg model:

The enviroment has 1-dim state and 1-dim action, the reward is calculated as follows:
- if state < 0.5, taking action < 0.5 gets reward 1, taking action >= 0.5 gets reward 0
- if state >= 0.5, taking action >= 0.5 gets reward 1, taking action < 0.5 gets reward 0

Train 500 iterations and test for 500 iterations:
If the average reward during test is larger than 0.9, this test passes

### CODE REVIEW

**Comment by yangdsh** [comment]
@dvanaken Test for dnn (Bohan's code) fails...

**Comment by yangdsh** [comment]
Perhaps my testing code and Bohan's testing code have some kind of dependency.
When one of our code is added/changed, the other one fails.
I modify Bohan's expectation values to pass the test.
I think it would be fine if no one else change our code in the future.

**Comment by bohanjason** [comment]
I guess it may be related to random seed.

**Comment by bohanjason** [comment]
I only set tensorflow random seed. Any you set numpy random seed in your test, which may also affect my test. 

I will reset all random seed in my test. It may solve the issue.

File: server/analysis/tests/test_nn.py [hunk 1]
```diff
@@ -29,10 +29,10 @@ def setUpClass(cls):
 
     def test_nn_ypreds(self):
         ypreds_round = ['%.3f' % x[0] for x in self.nn_result]
-        expected_ypreds = ['20.503', '22.158', '22.158', '25.692', '24.536', '23.637']
```
Comment (bohanjason) [inline server/analysis/tests/test_nn.py]: Could you please help me set both python seed and numpy seed before setting tensorflow seed (line 23) ? 
It should be something like this : 

        random.seed(0)
        np.random.seed(0)
        set_random_seed(0)

Then update the expected results accordingly. 
It should solve the dependency issue between our tests. 

Thanks.

Comment (yangdsh) [reply server/analysis/tests/test_nn.py]: No problem. FYI, there is a way to suggest changes. [https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request](url)

Comment (bohanjason) [reply server/analysis/tests/test_nn.py]: Thanks, Dongsheng.

**Comment by yangdsh** [comment]
@dvanaken The test fails because of connection broken, not the code issue.
I pushed an empty commit and it passes the test [https://travis-ci.org/cmu-db/ottertune/builds/597961056](url)

**Comment by dvanaken** [comment]
Thank you both for working on this!

**Pull request merged**